Update Agent Telemetry documentation (#26956)
DataDog/documentation: content/en/data_security/agent.md

{{< /tabs >}}

**Telemetry content:**

> To see up-to-date Telemetry content run the following command

Reviewer: ```suggestion
To view the latest Telemetry content, run the following command:

```

| ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
| **Checks**                                  |                                                                                                                        |
| checks.execution_time                       | Check's execution time in milliseconds                                                                                 |
| pymem.inuse                                 | Number of bytes allocated by the Python interpreter                                                                    |
| **Logs and metrics**                        |                                                                                                                        |
| dogstatsd.udp_packets_bytes                 | DogStatsD UDP packets bytes                                                                                            |
| dogstatsd.uds_packets_bytes                 | DogStatsD UDS packets bytes                                                                                            |

> | logs.auto_multi_line_aggregator_flush       | Number of multiline logs aggregated by the Agent                                                                       |

Reviewer: ```suggestion
| logs.auto_multi_line_aggregator_flush       | Number of multi-line logs aggregated by the Agent                                                                       |
```

| Metrics ([source][2])                       | Description                                                                                       |
| ------------------------------------------- | ------------------------------------------------------------------------------------------------- |
| **Checks**                                  |                                                                                                   |
| checks.execution_time                       | Check's execution time in milliseconds                                                            |
| pymem.inuse                                 | Number of bytes allocated by the Python interpreter                                               |
| **Logs and metrics**                        |                                                                                                   |
| dogstatsd.udp_packets_bytes                 | DogStatsD UDP packets bytes                                                                       |
| dogstatsd.uds_packets_bytes                 | DogStatsD UDS packets bytes                                                                       |

> | logs.bytes_missed                           | Total number of bytes lost before they could be consumed by the Agent, such as after log rotation |

Reviewer: ```suggestion
| logs.bytes_missed                           | Total number of bytes lost before they could be consumed by the Agent, for example, after log rotation |
```

> | logs.bytes_sent                             | Total number of bytes send before encoding, if any                                                |

Reviewer: ```suggestion
| logs.bytes_sent                             | Total number of bytes sent before encoding, if applicable                                                |
```

| logs.decoded                                | Total number of decoded logs                                                                      |
| logs.dropped                                | Total number of logs dropped                                                                      |

> | logs.encoded_bytes_sent                     | Total number of sent bytes after encoding, if any                                                 |

Reviewer: ```suggestion
| logs.encoded_bytes_sent                     | Total number of bytes sent after encoding, if applicable                                                 |
```

| postgres.get_new_pg_stat_activity_count     | Total rows fetched to collect `pg_stat_activity`                                                  |
| postgres.get_active_connections_ms          | Time to get active connections in milliseconds                                                    |
| postgres.get_active_connections_count       | Total rows fetched to get active connections                                                      |
| postgres.schema_tables_elapsed_ms           | Time to collect tables in Postgres schema                                                         |
| postgres.schema_tables_count                | Total tables in Postgres schema                                                                   |
| **API**                                     |                                                                                                   |
| api_server.request_duration_seconds         | CLI commands execution performance (if executed)                                                  |
| **Events**                                  |                                                                                                   |

> | agent_bsod                                  | Agent related BSOD info (very rare and only if Datadog Network Performace Monitoring is enabled)  |

Reviewer: ```suggestion
| agent_bsod                                  | Agent-related BSOD information (occurs only in rare conditions when Datadog Network Performance Monitoring is enabled)  |
```
